plasoscaffolder/templates/sqlite_plugin_test.jinja2

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""Tests for {{ plugin_name|replace('_',' ')}} plugin."""
+import unittest
+
+from plaso.lib import eventdata
+from plaso.lib import timelib
+from plaso.parsers.sqlite_plugins import {{ plugin_name }}
+
+from tests import test_lib as shared_test_lib
+from tests.parsers.sqlite_plugins import test_lib
+
+
+class {{ class_name }}Test(test_lib.SQLitePluginTestCase):
+  """Tests for {{ plugin_name|replace('_',' ')}} database plugin."""
+
+  @shared_test_lib.skipUnlessHasTestFile(['{{ database_name }}'])
+  def testProcess(self):
+    """Test the Process function on a {{ plugin_name|replace('_',' ')|title}} file."""
+    plugin_object = {{ plugin_name }}.{{ class_name }}Plugin()
+    storage_writer = self._ParseDatabaseFileWithPlugin(
+        ['{{ database_name }}'], plugin_object)
+
+    # We should have <NR> events in total.
```

Comment:
Please remove this comment, and add a test for the number of errors, see https://github.com/log2timeline/plaso/commit/2ab5131999e7cd5e8873eb0bf79c934a9a1c33bc for inspiration